run.py

```diff
@@ -1,11 +1,15 @@
 from flask import Flask
 from config import get_env_config
 from flask_migrate import Migrate
+from flask_cors import CORS
 
 
 def create_app(config_filename: str) -> Flask:
     app = Flask(__name__)
 
+    # setup for CORS policy
```

Comment:
Here's the [zulip link](https://anitab-org.zulipchat.com/#narrow/stream/237630-bridge-in-tech/topic/Troubleshoot/near/200494292) where I mentioned the steps I've done on bridge-in-tech-backend 😉

Reply:
```suggestion
    # setup for CORS policy
```
i too believe this way it's likely to face a problem in near future